resources/text/smallEvents/botFacts.json

```diff
@@ -28,33 +28,31 @@
 				"levelMean": "the last census noted that adventurers averaged **Level {infoNumber}**. Without knowing what to do with this information",
-				"nbMeanMoney": "the country bank accounted an average of **{infoNumber} :moneybag:** in its customers' bourse. Amused by this news,",
+				"nbMeanMoney": "the country bank accounted an average of **{infoNumber} :moneybag:** in its customers' savings. Amused by this news,",
 				"sumAllMoney": "the country bank accounted **{infoNumber} :moneybag:** in circulation at the moment. Surprised by this news,",
-				"richestPlayer": "the country bank evaluated that the richest adventurer currently has **{infoNumber} :moneybag:** in their account. Abandoned by this news,",
-				"trainedPets": "the SPAC, or the Counterfeiter Animal Protector, wishes to congratulate the trainers of the **{infoNumber} animals trained**. Impressive by this achievement,",
-				"feistyPets": "the SPAC, or Society for the Protection of Animals of the Land, campaigns to help ** {infoNumber} beastly animals ** to obtain a better life. Comforted by this news,",
-				"percentMalePets": "the SPAC, or the Counterfeiter Animal Protector, discovered that **{infoNumber}%** of the animals in the territory are masters. Happy with this news,",
-				"percentFemalePets": "the SPAC, or Society Protector of Treasured Animals, discovered that **{infoNumber}%** of the animals in the territory are females. Happy with this news,",
-				"guildLevelMean": "the AGC, or Counterfeit Guild Agency, noted that the current guilds averaged **Level {infoNumber}**. Intrigued by this information,",
-				"nbPlayersWithGivenClass": "according to a recent census of adventurers, there would be ** {infoNumber} ** with the class ** {infoComplement} **. Amazed by this news,",
-				"nbPlayersOnYourMap": "the surrounding knights counted ** {infoNumber} people ** currently circulating on your route. Surprised by this news,"
+				"richestPlayer": "the country bank evaluated that the richest adventurer currently has **{infoNumber} :moneybag:** in their account. Utterly lost on how to take the news,",
+				"trainedPets": "the ASS, the Animal's Safety Society, wishes to congratulate the owners of the **{infoNumber} animals trained**. Impressed by this achievement,",
+				"feistyPets": "the ASS, the Animal's Safety Society campaigns to help ** {infoNumber} beastly animals ** to obtain a better life. Comforted by this news,",
+				"percentMalePets": "the ASS, the Animal's Safety Society, discovered that **{infoNumber}%** of the animals in the territory are males. Pleased with this news,",
+				"percentFemalePets": "the ASS, the Animal's Safety Society, discovered that **{infoNumber}%** of the animals in the territory are females. Pleased with this news,",
+				"guildLevelMean": "the GA, the Guilds Association, noted that the current guilds averaged **Level {infoNumber}**. Intrigued by this information,",
```

Comment:
nous avons choisi un truc fictif pour la version française et du coup je serai chaud pr faire pareil en anglais tout en évitant de l'appeller "the ass" ptdr